Index: src/com/company/v2/FileNumberRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/company/v2/FileNumberRepository.java b/src/com/company/v2/FileNumberRepository.java
new file mode 100644
--- /dev/null	(date 1668957995356)
+++ b/src/com/company/v2/FileNumberRepository.java	(date 1668957995356)
@@ -0,0 +1,25 @@
+package com.company.v2;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileNumberRepository {
+    public int[] read() throws IOException { //todo: change the exception handling mechanism
+
+        //read the numbers from the text file
+        List<String> lines =  Files.readAllLines(Paths.get("C:\\Users\\Haridinu\\IdeaProjects\\Software_Construction\\src\\com\\company\\numbers.txt"));
+        String firstLine = lines.get(0);
+
+        //split first line by comma
+        String[] numbers = firstLine.split(","); //but don't use this kind of standard method it provides harm
+
+        //convert this String array to Integer array
+        int []  intNumbers = new int[numbers.length];
+        for(int i=0; i<numbers.length; i++) {
+            intNumbers[i] = Integer.parseInt(numbers[i]);
+        }
+        return intNumbers;
+    }
+}
