Index: src/com/company/v4/output/CommandLineOutput.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/company/v4/output/CommandLineOutput.java b/src/com/company/v4/output/CommandLineOutput.java
new file mode 100644
--- /dev/null	(date 1669298040723)
+++ b/src/com/company/v4/output/CommandLineOutput.java	(date 1669298040723)
@@ -0,0 +1,8 @@
+package com.company.v4.output;
+
+public class CommandLineOutput implements Output{
+    public void show(String message){
+
+        System.out.println(message);
+    }
+}
